python/cugraph/cugraph/community/louvain.py

```diff
@@ -72,9 +76,9 @@ def louvain(G, max_level=None, max_iter=None, resolution=1.0, threshold=1e-7):
         GPU data frame of size V containing two columns the vertex id and the
         partition id it is assigned to.
 
```

Comment:
You should use the defined column names below

Reply:
Changed to use defined column names